app/Jobs/SendReportbackToPhoenix.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Rogue\Jobs;
+
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Rogue\Services\Phoenix;
+use Rogue\Models\Reportback;
+
+class SendReportbackToPhoenix extends Job implements ShouldQueue
+{
+    use InteractsWithQueue, SerializesModels;
+
+    protected $reportback;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(Reportback $reportback)
+    {
+        $this->phoenix = new Phoenix;
+        $this->reportback = $reportback;
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $phoenix = new Phoenix;
```

Comment:
Definitely not necessary, but it may be cleaner to use the `$this->phoenix` that you created above rather than instantiating a second client as a local variable (or just remove the property above!).